controllers/articles.controller.js

```diff
@@ -0,0 +1,12 @@
+const { fetchArticles } = require("../models/articles.model");
+
+exports.getArticles = (req, res, next) => {
+  const { sort_by, order, topic } = req.query;
+  fetchArticles(sort_by, order, topic)
+    .then((articles) => {
+      res.status(200).send(articles);
```

Comment:
we **always** want to be sending our response back on an object with a descriptive key in line with JSON syntax. i.e. .send({articles: articles}) (the shorthand for which would be .send({articles}) )